src/pages/api/identity.tsx

```diff
@@ -0,0 +1,44 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { addIdentityToUser } from "../../command/addIdentityToUser";
+import { removeIdentityFromUser } from "../../command/removeIdentityFromUser";
+
+export default async (
+  request: NextApiRequest,
+  response: NextApiResponse,
+): Promise<void> => {
+  const { userId, type, value } = request.body;
+
+  if (request.method === "POST") {
+    return await addIdentityToUser({ userId, type, value })
+      .then(() => {
+        response.statusCode = 200;
+
+        return response.json("succeed");
```

Comment:
The custom is usually to return either the added thing (and that could help you don't need to refresh the page)

Reply:
Quick question, as the `response` object is massive. Do I have the data added in the `response` object ? Or do I, if succeed, return what was in the payload?

Reply:
I.e.
```ts
return await addIdentityToUser({ userId, type, value })
      .then((data) => {
        response.statusCode = 200;

        return response.json({data});
      })
```

Reply:
I think you may need some adjustments on that, depending on what is the return. 

Take care also that errors in a graphQL request are sent as a success in HTTP so I think we may need to check if there are errors in there

Reply:
> Take care also that errors in a graphQL request are sent as a success in HTTP so I think we may need to check if there are errors in there 

Yeah I realised that while building the app. Do you want me to make the change here? Maybe a new task would be wiser.

Reply:
Go for a new task